src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -214,8 +214,13 @@ public void amendHearing(CaseData caseData, String caseTypeId) {
                     for (DateListedTypeItem dateListedTypeItem
                             : hearingTypeItem.getValue().getHearingDateCollection()) {
                         var dateListedType = dateListedTypeItem.getValue();
+                        var listedDateWithZeroTime = dateListedType.getListedDate().split("T")[0] + "T00:00:00";
```

Comment:
var listedDateWithZeroTime = dateListedType.getListedDate().split("T")[0] + "T00:00:00";
Move this statement within the if block

Reply:
done